src/screens/Assets/Assets.js

```diff
@@ -191,7 +190,21 @@ class AssetsScreen extends React.Component<Props, State> {
           label: activeBNetworkTitle,
           action: () => navigation.navigate(ACCOUNTS),
           screenView: VIEWS.PPN_VIEW,
-          customHeaderButtonProps: { isActive: availableStake > 0 || hasUnsettledTx },
+          customHeaderButtonProps: {
+            isActive: availableStake > 0 || hasUnsettledTx,
+            backgroundColor: getColorByThemeOutsideStyled(theme.current, {
+              lightCustom: 'transparent', darkKey: 'synthetic140',
+            }),
+            color: getColorByThemeOutsideStyled(theme.current, {
+              lightKey: 'basic010', darkKey: 'basic090',
+            }),
+            style: {
+              borderWidth: 1,
+              borderColor: getColorByThemeOutsideStyled(theme.current, {
+                lightKey: 'basic005', darkKey: 'synthetic140',
+              }),
+            },
+          },
```

Comment:
In dark mode, the "Pillar Network" button is light blue, but looking at https://app.zeplin.io/project/5f4e5cdde148ba0c5d384e07/screen/5f8ebef41b3ae68326c38c08, it should match the light version.

Reply:
Hm, was following this approach: https://app.zeplin.io/project/5f4e5cdde148ba0c5d384e07/screen/5f68a097a3b27e18c2df0517.

Will check with the guys, which one is a way to go.

Reply:
Max updated PPN screens. 

(Some of them still show button similar to light mode's. Since this flow is planned to be updated, Max suggested keeping the same UI on different states).
So these changes are correct 🧑‍⚖️  😄